apps/pie-docs/src/components/assistive-text/code/props.json

```diff
@@ -23,6 +23,20 @@
           "default"
         ]
       }
+    ],
+    [
+      "isVisuallyHidden",
+      {
+        "type": "code",
+        "item": ["true", "false"]
+      },
+      "If true, hides the component visually but leaves it accessible for a11y technologies.",
+      {
+        "type": "code",
+        "item": [
+          false
```

Comment:
I'm surprised this works actually, and I prefer it to `"true"` and `"false"`. I won't ask you to change it anywhere else in this PR though!